adr/query.py

```diff
@@ -16,8 +16,12 @@
 from adr.cli import log
 
 here = os.path.abspath(os.path.dirname(__file__))
-
-ACTIVE_DATA_URL = 'http://activedata.allizom.org/query'
+if len(sys.argv)>2:
```

Comment:
You won't want to re-parse the raw sys.argv arguments (that's what the ArugmentParser object already did for us). Instead you'll need to use the `args.url` value that you'll get back in `cli.py`.

From there, we need to figure out a good way of propagating that value to `query.py`. One method might be to create a `set_active_data_url` function in `query.py` which changes the value of the global variable. We could call it after parsing the arguments.

Reply:
Thanks for your comments. I 've learned a lot :). I will check it!